expat/coverage.sh

```diff
@@ -227,7 +227,7 @@ _main() {
 
     echo 'Rendering HTML report...'
     _render_html_report "${coverage_dir}"
-    echo "--> ${coverage_dir}/index.html"
+    printf '--> %s/index.html\n' "${coverage_dir}"
```

Comment:
This looks more complex without added value other than a different take on style. I'll drop that but happy to re-add later if I was missing something important here.